app/src/lib/open-shell.ts

```diff
@@ -28,8 +28,7 @@ export function isGitOnPath(): Promise<boolean> {
 
   // adapted from http://stackoverflow.com/a/34953561/1363815
   return new Promise<boolean>((resolve, reject) => {
-    const options = { encoding: 'utf8' }
```

Comment:
No, `spawn` doesn't even take that option: https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options 🙈